src/style/index.scss

```diff
@@ -1,16 +1,17 @@
 @import "helpers";
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600');
 
 html, body {
```

Comment:
You're going to want to scope this differently so we don't change their page font.